docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
-  app-sqlalchemy:
-    container_name: stac-fastapi-sqlalchemy
+  stac-fastapi:
+    container_name: stac-fastapi
```

Comment:
We may want to continue to namespace this (`stac-fastapi-sqlalchemy`) in case people are developing on multiple backends simultaneously. I think it might cause confusion if the same container name is used for `stac-fastapi` core, `stac-fastapi-pgstac` and this repo.

Reply:
https://github.com/stac-utils/stac-fastapi-sqlalchemy/pull/1/commits/e86ea6a2595de9dfd154c8acaa01c7f14a5b8116